### PR TITLE
changes from Chris 12 11 2020

### PR DESCRIPTION
**Pull request by alex-novak**
1. Need to hide Sign up link on  /getting-started/ page, because it lead to the same page
2. change "Browse Jobs" button to be "Become a Contractor" and have that lead to Slack app
3. Browse Contractors - change to 'Get a Quote'
4. Change subtitle for 'Get a quote' to - "Get a free quote today and be working with your new WordPress developer in just a few hours after approval."
5. change "Find a WordPress Contractor" button to 'Get a Quote'(I assume with link)
6. Create 'Get a quote' page with MC form and add redirect from /pricing to /get-a-quote.

### CODE REVIEW

File: assets/getting-started.html [hunk 1]
```diff
@@ -66,7 +66,7 @@ <h2 class="pane-title"> To get started tell us what you are looking for. </h2>
                     </div>
                     <div class="field field-name-field-sign-link field-type-link-field field-label-hidden">
                       <div class="field-items">
-                        <div class="field-item even"><a href="https://talent.esteemed.io/jobs/#/" class="btn btn--blue" target="_blank">Browse Jobs</a></div>
+                        <div class="field-item even"><a href="http://esteemed.slack.com/" class="btn btn--blue" target="_blank">Become a Contractor</a></div>
```
Comment (albertvolkman) [inline assets/getting-started.html]: Please make this link https

File: assets/get-a-quote.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+title: Get a quote
+description: Get more details on WPcontractors pricing. We analyze your business and then craft a plan that's right for your needs.
+image: "/assets/img/pricing-plans.png"
+permalink: /get-a-quote/
+redirect_from:
+- /pricing/
+---
+{% include contact-form.html %}
```
Comment (albertvolkman) [inline assets/get-a-quote.html]: Missing EoL

Comment (alex-novak) [reply assets/get-a-quote.html]: fixed

File: assets/getting-started.html (hunk 1 above)
Comment (alex-novak) [reply assets/getting-started.html]: fixed

**Pull request merged**